packages/fxa-graphql-api/src/gql/dto/input/session-reauth.ts

```diff
@@ -14,5 +14,5 @@ export class SessionReauthInput extends SignInInput {
   public sessionToken!: hexstring;
 
   @Field()
-  public options!: SessionReauthOptionsInput;
+  public declare options: SessionReauthOptionsInput;
```

Comment:
Should this be `options!` instead of `options`?

Reply:
Nope, that will result in an error as `declare` doesn't need it.